sidekick/tests/test_models.py

```diff
@@ -31,3 +36,36 @@ def test_user_token_creation_no_duplicates(self):
         user.save()
 
         self.assertEqual(Token.objects.count(), 1)
+
+
+class ConsentModelTests(TestCase):
+    def test_url_generation(self):
+        """
+        Generates a URL that contains the code that reflects both the Consent and the
+        contact UUID
+        """
+        org = Organization.objects.create()
```

Comment:
We could probably move some of these into a setup function to keep things DRY

Reply:
Having a look at this again, I think the only things that we can move into a function are the org creation and the consent creation, unless we have a function that returns `consent, uuid, code`, which seems a bit messy.

Reply:
Oh ja, happy then to leave it as is.